python/graphstorm/config/argument.py

```diff
@@ -748,19 +752,72 @@ def eval_fanout(self):
             # By default use -1 as full neighbor
             return [-1] * self.num_layers
 
+    @property
+    def textual_data_path(self):
+        """ distillation textual data path
+        """
+        if hasattr(self, "_textual_data_path"):
+            return self._textual_data_path
+        return None
+
+    @property
+    def lm_name(self):
```

Comment:
Can you check if we can reuse the config.lm_model? (See https://github.com/awslabs/graphstorm/blob/main/python/graphstorm/config/argument.py#L148-L169)

Reply:
if we wanted to reuse it, we need to modify this code and define a new section of 
```
@property
    def distill_lm_configs(self):
```
This also adds new arguments and could be more confusing to customer cuz we put LM models for distillation and the LM models for node encoder under the same "lm_models" field. I feel the current implementation which put "lm_model" under the distillation is better. What's you suggestion on this?

Reply:
if we cannot settle the arguments in the short period of time (code freeze is friday), i suggest to remove `lm_name` and `pre_trained_name`. The current pipeline only supports one transformer model. Users don't need to specify them anyway in this release.

Reply:
This property can be removed